app/components/pair-compare.jsx

```diff
@@ -142,11 +143,13 @@ function PairCompare(props) {
     <div className={classes.container} {...otherProps}>
       <div className={classes.mainPointContainer}>
         <div className={classes.mainPointSubject}>{mainPoint.subject}</div>
+        <div style={{ marginBottom: '1rem'}}></div>
```

Comment:
It would be cleaner to just fix the margin of the mainPointSubject and mainPointDescription classes than to add extra divs.